qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -0,0 +1,253 @@
+    KUBE_ORGANIZATION: ((organization))
+    MAGIC_DNS_SERVICE: ((magic-dns-service))
+    GKE_PRIVATE_KEY_BASE64: ((gke-private-key-base64))
+    CAP_BUNDLE_URL: <%= bundle_url_from_version(enable_cf_deploy) %>
+  upgrade-task-params: &upgrade-task-params
+    << : *common-task-params
+    CAP_BUNDLE_URL: <%= bundle_url_from_version(upgrade_from_version) %>
+
+jobs:
+<% %w(SA HA).each do |avail| %>
+- name: ((pipeline-name))-<%= avail %>
+  plan:
+  - do:
+    - aggregate:
+      - get: ci
+      - get: s3.pg-sidecar
+        trigger: true
+      - get: s3.mysql-sidecar
+        trigger: true
+      - get: s3.scf-config-sles
+        trigger: true
+      - put: pool.kube-hosts
+        params: {acquire: true}
+        timeout: 2m
+    - task: cf-get-commit-id
+      file: ci/qa-pipelines/tasks/cf-get-commit-id.yml
+      input_mapping:
+        s3.archive: s3.scf-config-sles
+    <% if status_reporting %>
+    - put: status.src
+      params: &status-params-<%= avail %>
+        context: ((pipeline-name))-<%= avail %>
+        description: "QA Pipeline: ((pipeline-name)) (SLES <%= avail %>)"
+        path: commit-id/sha
+        state: pending
+    <% end %> # status_reporting
+    on_failure:
+      put: pool.kube-hosts
+      params: {release: pool.kube-hosts}
+  - do:
+    <% if upgrade_from_version %>
+    # This section only exists for upgrade pipelines; it determines how to set
+    # up the pre-upgrade deployment.
+    - task: cf-deploy-pre-upgrade
+      file: ci/qa-pipelines/tasks/cf-deploy.yml
+      params:
+        << : *upgrade-task-params
+        HA: <%= avail == 'HA' %>
+      input_mapping:
+        s3.scf-config: s3.scf-config-sles
+    <% if enable_cf_usb_upgrade_tests %>
+    - task: usb-deploy-pre-upgrade
+      file: ci/qa-pipelines/tasks/usb-deploy.yml
+      params:
+        GKE_PRIVATE_KEY_BASE64: ((gke-private-key-base64))
+    <% end %> # enable_cf_usb_upgrade_tests
+    <% if enable_cf_smoke_tests_pre_upgrade %>
+    - task: cf-smoke-tests-pre-upgrade
+      file: ci/qa-pipelines/tasks/run-test.yml
+      params:
+        << : *upgrade-task-params
+        TEST_NAME: smoke-tests
+      input_mapping:
+        s3.scf-config: s3.scf-config-sles
+    <% end %> # enable_cf_smoke_tests_pre_upgrade
+    <% if enable_cf_brain_tests_pre_upgrade %>
+    - task: acceptance-tests-brain-pre-upgrade
+      file: ci/qa-pipelines/tasks/run-test.yml
+      params:
+        << : *upgrade-task-params
+        TEST_NAME: acceptance-tests-brain
+      input_mapping:
+        s3.scf-config: s3.scf-config-sles
+    <% end %> # enable_cf_brain_tests_pre_upgrade
+    <% if enable_cf_acceptance_tests_pre_upgrade %>
+    - task: acceptance-tests-pre-upgrade
+      file: ci/qa-pipelines/tasks/run-test.yml
+      params:
+        << : *upgrade-task-params
+        TEST_NAME: acceptance-tests
+      input_mapping:
+        s3.scf-config: s3.scf-config-sles
+    <% end %> # enable_cf_acceptance_tests_pre_upgrade
+    <% if enable_cf_deploy %>
```

Comment:
enable_cf_upgrade

Reply:
But it's `enable-cf-deploy` in [`flags.yml`](https://github.com/SUSE/cf-ci/pull/248/files/5c05c5bcf031a6bfe03b100c61301f4f37a274b5#diff-b3614856767f8ba1e0bf96784e99bc18R31)?